tools/jdbc/src/test/java/org/duckdb/test/TestDuckDBJDBC.java

```diff
@@ -410,7 +411,63 @@ public static void duckdb_timestamp_test() throws Exception {
 
 		conn.close();
 	}
+
+	public static void test_duckdb_localdatetime() throws Exception {
+		Connection conn = DriverManager.getConnection("jdbc:duckdb:");
+		Statement stmt = conn.createStatement();
+		stmt.execute("CREATE TABLE x (ts TIMESTAMP)");
+
+		LocalDateTime ldt = LocalDateTime.parse("2021-01-18T21:20:07");
+
+		PreparedStatement ps1 = conn.prepareStatement("INSERT INTO x VALUES (?)");
+		ps1.setObject(1, ldt);
+		ps1.execute();
+		ps1.close();
+		
+		PreparedStatement ps2 = conn.prepareStatement("SELECT * FROM x");
+		ResultSet rs2 = ps2.executeQuery();
+
+		rs2.next();
+		assertEquals(rs2.getTimestamp(1), rs2.getObject(1, Timestamp.class));
+		assertEquals(rs2.getObject(1, LocalDateTime.class), ldt);
+
+		rs2.close();
+		ps2.close();
+		stmt.close();
+		conn.close();
+	}
 	
+	public static void test_duckdb_getObject_with_class() throws Exception {
+		Connection conn = DriverManager.getConnection("jdbc:duckdb:");
+		Statement stmt = conn.createStatement();
+		stmt.execute("CREATE TABLE b (vchar VARCHAR, bo BOOLEAN, sint SMALLINT, nint INTEGER, bigi BIGINT,"
+			+ " flt FLOAT, dbl DOUBLE, dte DATE, tme TIME, ts TIMESTAMP, dec16 DECIMAL(3,1),"
+			+ " dec32 DECIMAL(9,8), dec64 DECIMAL(16,1), dec128 DECIMAL(30,10))");
+		stmt.execute("INSERT INTO b VALUES ('varchary', true, 6, 42, 666, 42.666, 666.42,"
+			+ " '1970-01-02', '01:00:34', '1970-01-03 03:42:23', 42.2, 1.23456789, 987654321012345.6, 111112222233333.44444)");
+
+		PreparedStatement ps = conn.prepareStatement("SELECT * FROM b");
+		ResultSet rs = ps.executeQuery();
+
+		rs.next();
+		assertEquals(rs.getString(1), rs.getObject(1, String.class));
```

Comment:
Should we test the values as well as the types?

Reply:
The values should be equal to pass. assertEquals(a, b) basically calls a.equals(b) which typically should compare the values of a and b. 

E.g. this is false:
`LocalDateTime.of(1999,1,1,1,1).equals(LocalDateTime.of(2000,1,1,1,1))`

Reply:
Ah, I thought it was returning types for comparison. My bad.